src/base/http/server.cpp

```diff
@@ -56,13 +56,32 @@ namespace
     QList<QSslCipher> safeCipherList()
     {
         const QStringList badCiphers {u"idea"_qs, u"rc4"_qs};
+        // Contains Ciphersuites that use RSA for the Key Exchange but they don't mention it in their name
+        const QStringList badRSAShorthandSuites {u"AES256-GCM-SHA384"_qs, u"AES128-GCM-SHA256"_qs,
+                                                                       u"AES256-SHA256"_qs, u"AES128-SHA256"_qs,
+                                                                       u"AES256-SHA"_qs, u"AES128-SHA"_qs};
```

Comment:
The indentation looks too ugly.
```suggestion
        const QStringList badRSAShorthandSuites {
            u"AES256-GCM-SHA384"_qs, u"AES128-GCM-SHA256"_qs, u"AES256-SHA256"_qs,
            u"AES128-SHA256"_qs, u"AES256-SHA"_qs, u"AES128-SHA"_qs};
```

Reply:
Weird. I hadn't noticed the diff. The identation was different in QtCreator, closer to your suggestion. I'll try it again.